src/sciwrappers.h

```diff
@@ -39,9 +39,10 @@ sptr_t sci_send_message_internal (const gchar *file, guint line, ScintillaObject
 # endif
 #endif
 
-void 				sci_set_text				(ScintillaObject *sci,  const gchar *text);
-gboolean			sci_has_selection			(ScintillaObject *sci);
-void 				sci_end_undo_action			(ScintillaObject *sci);
+void                            sci_set_multipaste              (ScintillaObject *sci, gint mpval);
```

Comment:
This should be in the `GEANY_PRIVATE` section below.  Only plugin API functions should be outside of it.

Reply:
Again the reason I did this is because @elextr suggested that it should be an option.